### PR TITLE
Replace commands in the README with Makefile targets

### PR DESCRIPTION
**Pull request by chetan-rns**
**What type of PR is this?**

> /kind documentation

**What does this PR do / why we need it**:
Update the docs with the latest Makefile targets generated by operator-sdk

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by iam-veeramalla** [comment]
@chetan-rns I had to export the below 2 vars to my personal repos before running the make targets to create a new index image. The defaults for them is set to official quay and openshift.io registries.
IMAGE_TAG
IMAGE_TAG_BASE

Do you see a need for the documentation ?

File: README.md [hunk 1]
```diff
@@ -61,26 +67,36 @@ quay.io/redhat-developer/gitops-backend:v0.0.1
 If that's all what you are changing, the following steps are not needed in development
 mode. You could update your image "payload" and re-install the operator.
 
-* Build the operator image.
+Set the base image and version for building operator, bundle and index images.
+
+```
+export IMAGE=quay.io/redhat-developer/gitops-backend-operator VERSION=0.0.4
+```
```
Comment (chetan-rns) [inline README.md:74]: @iam-veeramalla I've updated the `Makefile` to work with a single image variable. Exporting this is sufficient for both operator and bundle images

Comment (iam-veeramalla) [reply README.md:74]: Thanks 🙏

**Comment by iam-veeramalla** [comment]
Verified the PR and it works fine.
/LGTM
/approve

**Pull request merged**